<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginNetSetWindow</class>
 <widget class="QWidget" name="LoginNetSetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>334</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>428</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginNetSetWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>68</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MYSQL配置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>100</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>98</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>138</y>
     <width>42</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>138</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>196</y>
     <width>65</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>REDIS配置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>231</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>231</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mysqlip">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>98</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mysqlport">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>98</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mysqlpasswd">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>138</y>
     <width>88</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mysqluser">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>138</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_redisport">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>305</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QToolButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>305</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QToolButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>231</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_redispasswd">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_redisip">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
